share/completions/kill.fish

```diff
@@ -4,10 +4,21 @@ for i in $__kill_signals
 	set number (echo $i | cut -d " " -f 1)
 	set name (echo $i | cut -d " " -f 2)
 	complete -c kill -o $number -d $name
-	complete -c kill -o $name -d $name
-	complete -c kill -o s -x -a \"$number\tSend\ $name\ signal\"
-	complete -c kill -o s -x -a \"$name\tSend\ $name\ signal\"
+	complete -c kill -o $name
+	complete -c kill -s s -x -a "$number $name"
```

Comment:
I still think having "number\t$name" and "$name" (without description) to be the way to go. The names are sort of descriptive (and we don't have a way to generate better descriptions) while the numbers aren't descriptive at all.